### ISSUE TITLE
Multiple initializations when used in multiple components

### ISSUE THREAD

**Issue by cedrichaase** [meiblorn/ngx-fullpage#306]
Hey there,

first of all, thanks a lot for developing this project!
I have tried to integrate the mnFullpage directive with an application that uses it in multiple components. When navigating between those components, the second time the directive is encountered (and on all subsequent encounters), fullPage.js throws the following error message and functionality breaks:
```
fullPage: Fullpage.js can only be initialized once and you are doing it multiple times!
```

I have created a PR that fixes this by destroying fullpage every time before initializing it. If you could accept the PR or offer any other kind of help/workaround for this issue in case I'm missing something, I would appreciate it a lot :)

**Comment by ybarbaria** [comment]
Hello, 

I have the same problem. 
@cedrichaase Do u think your fix will be merge soon ? 

Thx

**Comment by jpvignolo** [comment]
Same issue here. Actually it also occurs also when you navigate from a fullscreen component to a regular component and then back to a fullscreen component.
A quick fix is to do destroy your fullpage service in the ngOnDrestoy lifecycle method :
```typescript
import {
  Component,
  OnInit,
  OnDestroy
} from '@angular/core';

import { MnFullpageService } from 'ng2-fullpage';

export class YourFullpageComponent implements OnInit {
[...]
  constructor( private fullpageService: MnFullpageService ) {}
[...]
  ngOnDestroy() { this.fullpageService.destroy('all'); }
[...]
}
```

**Comment by ybarbaria** [comment]
Thx @jpvignolo  good job 👍

**Comment by cedrichaase** [comment]
Thanks for contributing the workaround @jpvignolo !
Unfortunately, this method does not do the trick for me, as I'm working with a multitude of components and I don't want to have to implement `OnDestroy` for every one of them just for the purpose of making fullpage.js work.

Just in case anyone stumbles upon this thread looking for a cleaner solution: I also tried calling the `destroy` method every time the angular Router's `NavigationStart` event is triggered, which didn't seem to work for some reason though.

As such, I had to abandon the idea of using this library for now. Seeing as this is a pretty cool project and I'm not the only one who experiences this issue, I'd still be happy to get a comment by @meiblorn on this and help getting a proper solution on the way!
